src/app/cheatsheets/aliasing/aliasing.component.html

```diff
@@ -231,10 +231,15 @@ <h3>Cvar Table</h3>
         <td>int</td>
       </tr>
       <tr>
-        <td><code>currentHp</code></td>
-        <td>(scripting only) The character's current remaining HP.</td>
+        <td><code><del>currentHp</del></code></td>
+        <td><del>(scripting only) The character's current remaining HP.</del> This has been deprecated, use <code>get_hp()</code> instead.</td>
```

Comment:
Technically, it was deprecated a long time ago and has now been completely removed - this change implies that it will still work. This row should probably be entirely deleted instead.